### PR TITLE
GeCraft auto-height for cards

### PR DESCRIPTION
**Pull request by ancientTexts-net**
# Describe what your pull request addresses

- [ ]

## Test Instructions

- [ ]

TODO @ancientTexts-net

### CODE REVIEW

File: src/hooks/useWindowSize.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    function updateSize() {
+      setSize([window.innerWidth, window.innerHeight]);
+    }
+
+    window.addEventListener('resize', updateSize);
+    updateSize();
+    return () => window.removeEventListener('resize', updateSize);
+  }, []);
+  return size;
+};
```
Comment (PhotoNomad0) [inline src/hooks/useWindowSize.js:15]: Good find here.

File: src/components/Workspace/index.js [hunk 2]
```diff
@@ -126,6 +158,7 @@ Workspace.propTypes = {
     xxs: PropTypes.number,
   }),
   rowHeight: PropTypes.number,
+  rows: PropTypes.number,
```
Comment (PhotoNomad0) [inline src/components/Workspace/index.js:161]: Suggest adding comments on these new properties - such as what they do for the layout.

I wish the original author had commented the original properties, but going forward we should add comments as new properties are added.

**Review by PhotoNomad0 — CHANGES_REQUESTED** [review]
Code looks good to me. Needs a little more documentation.

**Comment by PhotoNomad0** [comment]
Styleguidist example looks good.  Will test with GatewayEdit for compatibility.

File: src/hooks/useWindowSize.js (hunk 1 above)
Comment (mannycolon) [reply src/hooks/useWindowSize.js:15]: Great good! 
bitmoji

**Pull request closed without merging**